concepts/structs/about.md

```diff
@@ -39,6 +41,37 @@ fmt.Printf("name: %s age: %d\n", person.name, person.age)
 // Output: name: age: 0
 ```
 
+Another way of creating a new instance of a struct is by using the `new` built-in: 
+```go
+s2 := new(Shape)
+fmt.Printf("name: %s size: %d\n", s2.name, s2.size)
+// Output: name:  size: 0
+```
+
+Sometimes it can be nice to have a _constructor_, which is a function to create an instance of a struct.
+The following constructor can be used to create a new instance of `Shape` and automatically set a default-value for the size of the shape:
+
+```go
+func NewShape(name string) Shape {
+	return Shape{
+		name: name,
+		size: 100, //default-value for size is 100
+	}
+}
+```
+
+Using a constructor can have the following advantages:
+* validation of the given values
+* handling of default-values
+
+Another advantage will be shown with an example. When importing the `package shapes` containing the
+struct `Shape` the unexported fields `name` and `size` cannot be accessed. Field names in structs follow the Go convention - fields whose name starts
+with a lower case letter are only visible to code in the same package, whereas those whose name starts with an upper case letter are visible in other packages.
+Using a constructor the initial value can nevertheless be set:
+```go
+rect := shapes.NewShape("Rectangle")
+```
+
 You can create an instance of a `struct` without using the field names, as long as you define the fields _in order_:
```

Comment:
This bit makes more sense to be together with the part where we talk about how to initialize structs using struct literals. After showing how to initialize the struct with the fields, we can say that including the name of the fields is optional.